xml/System.Security.Cryptography.Pkcs/SignedCms.xml

```diff
@@ -926,8 +927,8 @@ The following permissions are required to access the signature key:
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="certificate">The certificate to remove from the collection.</param>
+        <summary>Removes the specified certificate from the collection of certificates for the encoded CMS/PKCS #7 message, if it exists.</summary>
         <remarks>To be added.</remarks>
```

Comment:
The source shows that the method throws a CryptographicException if the certificate is not found: 
![image](https://user-images.githubusercontent.com/10886961/62567114-66e14300-b83f-11e9-830f-73e9ef231047.png)